shuffle/cli/src/main.rs

```diff
@@ -130,7 +149,9 @@ fn normalized_address(account_address: Option<String>) -> Result<String> {
                 ));
             }
             let address = fs::read_to_string(home.get_latest_address_path())?;
-            Ok("0x".to_owned() + &address)
+            Ok(AccountAddress::from_hex_literal(
+                ("0x".to_owned() + &address).as_str(),
```

Comment:
Let's refactor this method and pull a lot of the functionality out of the `match` block. Let's have the match block solely focus on getting a readable string, and then have AccountAddress work afterwards:

```rust
let normalized_str = match account_address {
  Some(address) => { if &address[0..2] != "0x" { "0x" + address } else { address } },
  None => fs::read_to_string(home.get_latest_address_path())?
}
Ok(AccountAddress::from_hex_literal(normalized_str))

```

Reply:
Great idea! This will help make it more readable. Will do this